src/ess_rhat.jl

```diff
@@ -565,7 +569,7 @@ function _ess_rhat(
         ess[i] = min(ntotal / τ, ess_max)
     end
 
-    return ess, rhat
+    return (ess=ess, rhat=rhat)
```

Comment:
A bit simpler:
```suggestion
    return (; ess, rhat)
```

Reply:
Ah, I had forgotten this was already supported on v1.6.